src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.util;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+public class HibernateUtil {
+
+    private HibernateUtil() {
+    }
+
+    public static SessionFactory getSessionFactory() {
+        EntityManagerFactory emf = Persistence.createEntityManagerFactory("mate-academy");
+        EntityManager entityManager = emf.createEntityManager();
+        // Get the Hibernate Session from the EntityManager in JPA
+        Session session = entityManager.unwrap(org.hibernate.Session.class);
+        return session.getSessionFactory();
+    }
```

Comment:
Let's use either hibernate or persistence configuration here. And let's create only one instance of SessionFactory/EntityManagerFactory in init method, and in method getSessionFactory() will return it?